CONTRIBUTING.md

```diff
@@ -1,16 +1,281 @@
+[new-feature-request]: https://github.com/RIOT-OS/RIOT/issues/new?template=feature_request.md&title=Feature+Request:
+
+## Bug Reports
+[bug-reports]: #bug-reports
+
+While bugs are unfortunate, they're a reality in software. We can't fix what we
+don't know about, so please report liberally. If you're not sure if something
+is a bug or not, feel free to file a bug report anyway.
+
+**If you believe reporting your bug publicly represents a security risk to
+RIOT users, please send an email describing the bug to security@riot-os.org**.
+We would appreciate waiting for a 6 months grace period before reporting it on
+public channels, to allow us adequate time to release the fix.
+
+Before reporting a bug, have a look at [opened bugs][opened-bugs-link]. Maybe
+someone has already reported your error.
+
+Once you have verified that they haven't, opening an issue is as easy as
+clicking on [this link][bug-report-link] and filling out the fields.
+
+[opened-bugs-link]: https://github.com/RIOT-OS/RIOT/issues?q=state:open+type:issue+label:bug
+[bug-report-link]: https://github.com/RIOT-OS/RIOT/issues/new?template=bug_report.md&title=Bug:
+
+Each bug report issue uses a template with 5 sections that are there to help
+other contributors understand your issue and eventually reproduce it:
+
+    #### Description
+
+    #### Steps to reproduce the issue
+
+    #### Expected results
+
+    #### Actual results
+
+    #### Versions
+
+To fill the `Versions` section, you can use the script provided in the RIOT git
+repository:
+```
+./dist/tools/ci/print_toolchain_versions.sh
+```
+
+In summary, try to include as much information as possible, to help maintainers
+or other developers to fix the bug quickly.
+
+## Pull Requests
+[pull-requests]: #pull-requests
+
+GitHub's Pull Request (PR) feature is the primary mechanism used to make
+contributions to the RIOT codebase. GitHub itself has some great documentation
+on [using the Pull Request feature][about-pull-requests].
+We use the [fork and pull model][development-models], where contributors push
+changes to their personal fork and create pull requests to bring those changes
+into the source repository.
+
+[about-pull-requests]: https://help.github.com/articles/about-pull-requests/
+[development-models]: https://help.github.com/articles/creating-a-pull-request-from-a-fork
+
+### General rules
+
+* Before opening a new Pull Request, have a look at
+  [existing ones][existing-pull-requests]. Maybe someone has already opened one
+  about the same thing. If it's the case, you might be able to help with the
+  contribution. Just comment on the PR and ask.
+  Old and stalled [PRs are sometimes archived][archived-pull-requests] with a
+  "memo" label, maybe one of them is also about the same topic.
+
+* Each Pull Request form uses a template with 3 sections that are here to help
+  maintainer understand your contribution and help them testing it:
```

Comment:
either "maintainers" or "a maintainer" (but since you use "them" later on, and the singular "they" seems to be confusing for some people maybe rather use "maintainers" ;-)).